core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -474,7 +474,7 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 			}
 
 			// Add the name of the submodule
-			$this->Template->headline .= ' <span>' . sprintf($GLOBALS['TL_LANG'][$strTable][Input::get('key')][1], Input::get('id')) . '</span>';
+			$this->Template->headline .= ' <span>' . sprintf($GLOBALS['TL_LANG'][$strTable][Input::get('key')][1] ?? '%s', Input::get('id')) . '</span>';
```

Comment:
This does not make sense because it will turn this

<img width="596" alt="" src="https://user-images.githubusercontent.com/1192057/152835779-7babbffc-626c-49dd-bf8b-04dbe81a3c40.png">

into this:

<img width="623" alt="" src="https://user-images.githubusercontent.com/1192057/152835813-629c9c70-19b9-4e5c-86a1-be20b4fbad98.png">

Maybe we should wrap the line with an `if (isset($GLOBALS['TL_LANG'][$strTable][Input::get('key')][1]))` instead?

Reply:
That would be wrong. The label is supposed to say what the custom action does and not the ID of the record.

Reply:
Fallback to `Input::get('key')` makes most sense IMO.

Reply:
adjusted in  e02372fcd9376dc91a9d4c48fb8c66155e06c957